### DISCUSSION TITLE
Ghostfolio looks great!

### DISCUSSION THREAD

**Discussion by ups7** [ghostfolio/ghostfolio#2127] [General]
Hello!

I wanted to say that Ghostfolio looks great! I have installed a self-hosted version and been playing with it for a few days, and have found it to be just what I was looking for. Thanks to everyone who has contributed to this project!

One question: I have some ideas/suggestions to add features or improve some things but I am not a developer and will not be able to implement them. I don't want to seem like just asking for things without contributing back, so my question is if I should still post the ideas?

Keep up the great work!

**Comment by dtslvr** [comment]
Hi @ups7

Thank you very much for your kind words about Ghostfolio! I'm glad to hear that you are enjoying the self-hosted version and that it meets your needs.

We appreciate your ideas and suggestions, even if you're not a developer. Please join our Slack community if you have not yet where we can discuss your ideas and decide the best way to proceed, e.g. creating detailed GitHub tickets.

Thanks again and we look forward to collaborating with you to enhance this project!

Kind regards
Thomas
